app/assets/stylesheets/new.scss

```diff
@@ -1,3 +1,5 @@
+@import 'foundation_and_overrides.scss';
```

Comment:
Imported partial `foundation_and_overrides.scss` should be written as `foundation_and_overrides`<br>Prefer double-quoted strings